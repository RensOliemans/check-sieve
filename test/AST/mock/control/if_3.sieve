require "fileinto";

if true {
    keep;
    stop;
} elsif false {
    fileinto "\\Seen";
    stop;
} else {
    stop;
}