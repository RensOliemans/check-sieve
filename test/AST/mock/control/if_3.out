Mail Sieve
    Require
        String ("fileinto")
    Branch
        Condition
            True
        Block
            Command (keep)
            Command (stop)
        Condition
            False
        Block
            Command (fileinto)
                String ("\\Seen")
            Command (stop)
        Block
            Command (stop)
